src/routes/controllers/users.controller.ts

```diff
@@ -66,10 +66,10 @@ export const getAllGrades = async (req: Request, res: Response) => {
 };
 
 export const deleteUserById = async (req: Request, res: Response) => {
-  const id = req.params.id;
+  const { id } = req.params;
```

Comment:
qua si prende in automatico l'id? c'è solo questo parametro sui params?

Reply:
c'è solamente lui come parametro